client-java/controller/src/test/java/org/evomaster/client/java/controller/internal/db/h2/H2SchemaExtractorTest.java

```diff
@@ -189,7 +192,7 @@ public void testColumnUpperBoundConstraint() throws Exception {
         assertEquals("INTEGER", columnDto.type);
 
         assertEquals(1, fooTable.tableCheckExpressions.size());
-        assertEquals("(AGE_MAX <= 100)", fooTable.tableCheckExpressions.get(0).sqlCheckExpression);
+        assertEquals("(\"AGE_MAX\" <= 100)", fooTable.tableCheckExpressions.get(0).sqlCheckExpression);
```

Comment:
what was the problem with the escaping of these variables?

Reply:
They started appearing in the latest version of H2.